packages/markdown/remark/src/index.ts

```diff
@@ -103,6 +96,15 @@ export async function createMarkdownProcessor(
 		...remarkRehypeOptions,
 	});
 
+	if (!isPerformanceBenchmark) {
+		// Syntax highlighting
+		if (syntaxHighlight === 'shiki') {
+			parser.use(rehypeShiki, shikiConfig);
+		} else if (syntaxHighlight === 'prism') {
+			parser.use(rehypePrism);
+		}
```

Comment:
You could change the order here. I.e. move these plugins before or after user provided rehype plugins.

Reply:
It looks like this code is already moved before user-defined rehype plugins, but the [other comment](https://github.com/withastro/astro/pull/10104#discussion_r1488274899) was after.